src/main/scala/mesosphere/marathon/raml/AppConversion.scala

```diff
@@ -308,6 +303,20 @@ trait AppConversion extends DefaultConversions with ConstraintConversion with En
       ))
       else None
 
+    val unreachableStrategy = service.when(_.hasUnreachableStrategy, _.getUnreachableStrategy.toRaml).orElse(App.DefaultUnreachableStrategy)
+
+    val hasPersistentVolumes = service.hasContainer && service.getContainer.getVolumesList.exists(_.hasPersistent)
```

Comment:
Where is this coming from?

Reply:
That's how we can find if we have persistent volumes. You can check the persistent volumes field in the RunSpec for more details.